### PR TITLE
Veoss 979

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

**Comment by oliwiaKot** [comment]
retest this please

**Comment by oliwiaKot** [comment]
retest this please

**Comment by oliwiaKot** [comment]
retest this please

File: app/models/domain/EuTaxRegistration.scala [hunk 1]
```diff
@@ -62,7 +62,8 @@ object RegistrationWithFixedEstablishment {
 
 final case class RegistrationWithoutFixedEstablishment(
                                     country: Country,
-                                    taxIdentifier: EuTaxIdentifier
+                                    taxIdentifier: EuTaxIdentifier,
+                                    sellsGoods: Option[Boolean]
```
Comment (DevAndyroo) [inline app/models/domain/EuTaxRegistration.scala:66]: I think this should be sendsGoods

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/models/domain/EuTaxRegistration.scala (hunk 1 above)
Comment (oliwiaKot) [reply app/models/domain/EuTaxRegistration.scala]: oh yes! thanks, just renamed it

**Comment by oliwiaKot** [comment]
retest this please

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**